.travis.yml

```diff
@@ -21,7 +21,7 @@ before_install:
 install:
 - npm install
 - pip install --user -r requirements.txt
-- if [ "$CXX" = "g++" ]; then export CXX="g++-4.9" CC="gcc-4.9"; fi
+- if [ "$CXX" = "g++" ]; then export CXX="g++-4.9" CC="g++-4.9 -std=c++11"; fi
```

Comment:
The support we need is actually just in a header file guarded by #if __cplusplus >= 201103, but nothing in that code requires an underlying C++11 compiler feature, it's simply part of a newer stdlib standard.  We could extract the parts we need into Vireo and use it directly... if there aren't license issues with doing that.